src/dmd/dclass.d

```diff
@@ -746,6 +746,13 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
         FuncDeclaration fdmatch = null;
         FuncDeclaration fdambig = null;
 
+        void updateBestMatch(FuncDeclaration fd)
```

Comment:
Since you use the context variables `fdmatch` and `fdambig` it could be declared without parameter at all, at line 763.

Reply:
True, although I thought it was more clear to make the updated value explicit.  I'm willing to change it, but prefer this version.